lib/opensusebasetest.pm

```diff
@@ -336,11 +333,6 @@ sub wait_boot {
             # harddisk' is above
             send_key_until_needlematch 'inst-bootmenu-boot-harddisk', 'up';
             boot_local_disk;
-            if (check_var('ARCH', 'aarch64') and get_var('UEFI')) {
```

Comment:
ah it's just moved, but boot_local_disk was made for ppc64le, not sure if this is needed